source/class/qx/ui/table/pane/Scroller.js

```diff
@@ -697,17 +697,13 @@ qx.Class.define("qx.ui.table.pane.Scroller",
     {
       this.__tablePane.onTableModelDataChanged(firstRow, lastRow, firstColumn, lastColumn);
       var rowCount = this.getTable().getTableModel().getRowCount();
-      var colCount = this.__table.getTableColumnModel().getOverallColumnCount();
 
       if (rowCount != this.__lastRowCount)
       {
         this.updateVerScrollBarMaximum();
-
-        if (this.getFocusedRow() === null && rowCount > 0 && colCount > 0)
-        {
-          this.setFocusedCell(this.getFocusedColumn()||0, 0);
-        }
-        else if (this.getFocusedRow() >= rowCount)
+        
+        const focusedRow = this.getFocusedRow();
+        if (focusedRow !== null && focusedRow >= rowCount)
```

Comment:
I'm not quite following this change, but it looks to me not to be equivalent. What is the purpose of this change?

Reply:
It is not meant to be equivalent. I gave a short explanation [here:](https://github.com/qooxdoo/qooxdoo/pull/10277#issuecomment-970145759)

> I am sorry for the multiple small commits. Forgot something everytime. I added 2 aria attrs to the table and changed the focus handling a bit. It was possible that the moveFocusedCell calls from pressing keyboard keys didn't went through when there was no focused cell before. This was strange because the table showed a focus. Turns out the Scroller set the focus to the first cell, but the table didn't. I found that handling a little bit weird for keyboard only users. The cell didn't get announced when the table got focus because of that too.
> 
> I removed the first cell focus from the Scroller and changed the moveFocusedCell so that the first possible cell gets focused if there was no cell focus before